Virtual Garage Fixes

Riverman02 opened the pull request:
**When merged this pull request will:**
- Temporary fix for Cockpit Doors not hiding. Doors can now be individually hidden
- Forces the variants to show in the garage ~(will show duplicates)~
- Removes the ability to add/remove parts between variants. Leaves common customization across the 3 base variants. (In-game removal remains unchanged, this only affects Garage)

Riverman02 committed:
Separate Door Hide/Unhide

Riverman02 committed:
Removed Erroneous Hide Garage option

Riverman02 committed:
Garage customization options removed

Riverman02 committed:
Force Variants in Garage

BroBeansCPG commented on addons/uh60_config/config/CfgAnimationSources.hpp:
```diff
@@ -232,40 +232,51 @@ class AnimationSources: AnimationSources {
     mass = MASS_MH60M_MISC;
   };
   class Fuelprobe_show {
-    displayName = "Show Fuel Probe";
```
Is removing this option intentional? I'd think it'd make sense to keep the option to show and hide the fuel probe and ERFS from the MH/Pavehawk versions

Riverman02 replied:
From my testing, enabling it still enables the option for the UH variants

Riverman02 committed:
Remove Duped Garage variants

Riverman02 committed:
Removed Mission and OPFOR Variants

Features have been neglected for awhile, we can look into adding something like this back when other higher priority objectives have been met.

Riverman02 committed:
Added Cabin Doors to Garage Options
